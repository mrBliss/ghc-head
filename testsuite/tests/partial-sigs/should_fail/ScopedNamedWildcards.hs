{-# LANGUAGE NamedWildcards #-}

module ScopedNamedWildcards where

test2 :: _a -> _a
test2 x = let y :: _a
              y = (x,undefined)
          in y
