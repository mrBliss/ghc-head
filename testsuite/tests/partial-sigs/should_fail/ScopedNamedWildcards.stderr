
ScopedNamedWildcards.hs:9:20:
    Occurs check: cannot construct the infinite type: t ~ (t, t1)
    Relevant bindings include
      y :: (t, t1) (bound at ScopedNamedWildcards.hs:9:15)
      x :: (t, t1) (bound at ScopedNamedWildcards.hs:8:7)
      test2 :: (t, t1) -> (t, t1) (bound at ScopedNamedWildcards.hs:8:1)
    In the expression: x
    In the expression: (x, undefined)
